### ISSUE TITLE
Facebook App Needs to be Live

### ISSUE THREAD

**Issue by pogzie** [jprjr/multistreamer#40]
I just noticed something. I've properly configured my Facebook, Twitch and YouTube keys and secrets and everything was going well. 'Til I tried visiting my pages as incognito and found that Twitch and YouTube shows the stream/videos as is but Facebook, well its a bit sketchy. 

As an admin of my own page, I see my live/previous streams without any problems. I always wondered about the question mark but did not bother with it. 

![Wat](https://i.imgur.com/kgqhElb.png)

So I went on with my life and streamed with Multistreamer some more. I checked in incognito my public FB page and lo and behold, nothing was friggin there. I've been streaming with Multistreamer since January and the last post was December, probably using NVidia or Restream. 

![Old](https://i.imgur.com/Gu2xu55.png)

Then I did an experiment, I created a Privacy Policy page and published and made my app live. There it goes, I saw every Multistream post even in incognito. Even a random guy in some remote corner of the world, stumbles upon my FB page, he can see the full glory of my noobidity. 

![Live](https://i.imgur.com/aBph0uB.png)

TLDR; please include in the FB guide that the app needs to go live. That involves a Privacy Policy link being published and flipping the app switch to live. Then again, you will still have that question mark even if its live, but check your page in incognito to make sure its published publicly in FB.

**Comment by jprjr** [comment]
I think the Privacy Policy only matters when you want the general public to be able to use the app. My production instance is set to "Live", but I haven't written a privacy policy or submitted the app to Facebook Review, I just have a small handful of friends that I gave the "Tester" role.

But you're right, that should be mentioned in the admin guide - adding it now.

**Comment by jprjr** [comment]
And done - https://github.com/jprjr/multistreamer/wiki/Facebook

**Comment by pogzie** [comment]
Thanks for taking note.  Its more like: if the app is not live, your streams will only be viewed by the admin of the page, it will not appear in any other peoples walls or on the page itself when viewed by any other user or publicly. 

I dont share my multistreamer with anyone but I was just surprised that when i checked my FB page in incognito, all my multistreamer videos were not there (when the app was not live). Flipping the switch to live would show all the previous videos though.

**Comment by pogzie** [comment]
Necessary documentation already provided in the FB section of the readme. Closing.
